packages/prop-controllers/src/gap-x/gap-x.ts

```diff
@@ -1,16 +1,21 @@
 import { Types } from '../prop-controllers'
 import { GapData, responsiveGapDataSchema } from '../data'
-import { versionedPropDef } from '../versioned'
+import { versionedPropDef, typeArg } from '../versioned'
 
-export const GapX = versionedPropDef<{
-  preset?: unknown
-  label?: string
-  defaultValue?: GapData
-  min?: number
-  max?: number
-  step?: number
-  hidden?: boolean
-}>()(Types.GapX, responsiveGapDataSchema, {
-  version: 1,
-  dataKey: 'prop-controllers::gap-x::v1',
-})
+export const GapX = versionedPropDef(
```

Comment:
Here and below, adjusting existing prop controller definitions for the revised `versionedPropDef` API, which takes options wrapped in `typeArg`.